client/my-sites/domains/domain-management/edit-contact-info-page/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
+import Main from 'calypso/components/main';
+import BodySectionCssClass from 'calypso/layout/body-section-css-class';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import isRequestingWhois from 'calypso/state/selectors/is-requesting-whois';
+import type { EditContactInfoPageProps } from './types';
+
+const EditContactInfoPage = ( props: EditContactInfoPageProps ): JSX.Element => {
```

Comment:
The linter complains about `props`, since it's not used. But I say we should still keep the type declaration and only omit the `props` variable...

Reply:
Yep, removed it for now.